lib/LaTeXML/Package/actuarialangle.sty.ltxml

```diff
@@ -0,0 +1,36 @@
+# -*- mode: Perl -*-
+# /=====================================================================\ #
+# |  actuarialangle                                                     | #
+# | Implementation for LaTeXML                                          | #
+# |=====================================================================| #
+# | Part of LaTeXML:                                                    | #
+# |  Public domain software, produced as part of work done by the       | #
+# |  United States Government & not subject to copyright in the US.     | #
+# |---------------------------------------------------------------------| #
+# | Bruce Miller <bruce.miller@nist.gov>                        #_#     | #
+# | http://dlmf.nist.gov/LaTeXML/                              (o o)    | #
+# \=========================================================ooo==U==ooo=/ #
+package LaTeXML::Package::Pool;
+use strict;
+use warnings;
+use LaTeXML::Package;
+
+InputDefinitions('actuarialangle', type => 'sty', noltxml => 1);
+
+DefConstructor('\actuarialangle{}',
+  "<ltx:XMApp>"
+    . "<ltx:XMTok role='ENCLOSE' enclose='actuarial' meaning='actuarialangle' />"
+    . "<ltx:XMWrap>#1</ltx:XMWrap>"
+    . "</ltx:XMApp>");
+
+# define \overanglebracket, unless the nobracket option is passed
+if (IfCondition('\ifacta@bracket')) {
+  DefMath('\overanglebracket{}', "\x{23E0}", operator_role => 'OVERACCENT'); # TOP TORTOISE SHELL BRACKET
+  DefMath('\group{}', '\overanglebracket{#1}');
+}
+
+# preserve reversion
+Let('\lx@actuarialangle@angl', '\angl');
+DefMath('\angl{}', '\lx@actuarialangle@angl{#1}', role => 'ENCLOSE');
```

Comment:
So I have decided that preserving the reversions of `\angln` and others is overkill. However, the TeX reversion of `\angln` comes out as `{\angl{n}}` instead of just `\angl{n}`. Also, if I use `\angln`, the `<Math>` tag loses the `text` attribute. I find this a bit odd.

Reply:
Ah, my mistake: Omit the `role=>'ENCLOSE'` from the `DefMath`; that says that the whole angl expression is an ENCLOSE, which it isn't and messes up parsing, so you lose the `text` attribute.

The extra braces on `\angln` are because that's how the style defines it; To the extent that `\angln` is a Thing, you could define it (and its friends) simply with
```
DefMath('\angln','\angl{n}');
```
and you could even add specific properties (like meaning) to it, if you wanted.

Reply:
Uh, I don't remember why I added `role => 'ENCLOSE'` anymore – I was making too many small experiments. It was not in your original suggestion, and I removed it. Now the `text` attribute works. I guess I would have seen a warning in the log? I am still getting used to the new error reporting style.

And indeed, `\angln` is defined as `{\angl n}`. I replaced them with `DefMath`'s.